graphql-dgs-spring-boot-micrometer/src/test/kotlin/com/netflix/graphql/dgs/metrics/micrometer/MicrometerServletSmokeTest.kt

```diff
@@ -242,6 +242,32 @@ class MicrometerServletSmokeTest {
             )
     }
 
+    @Test
```

Comment:
I don't see any metrics being verified here? I understand the change on its own, but how is it related to Micrometer other than being used in this test?

Reply:
I could verify the metrics here, however there is a similar test with literally the same metrics above.
This is not directly related to Micrometer; it would be better to move such a test to the main module.
I put it here just to cut the corner of initialization of a wrapped DataLoader

Reply:
Yup, sure. I also found out that tests within DgsDataLoaderInstrumentationTest don't verify anything, because verifications made within `whenComplete` and run aside main thread
Would you like me to fix it?